app/graphql/types/artist_type.rb

```diff
@@ -14,10 +14,20 @@ class ArtistType < Types::BaseObject
     field :created_at, GraphQL::Types::ISO8601DateTime, null: false
     field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
 
-    field :posts, [Types::PostType], null: false
+    field :posts, [Types::PostType], null: false do
+      argument :orderBy, PostOrderEnum, required: false
+    end
+
+    def posts(orderBy: nil)
```

Comment:
Why is `orderBy` given a default value here? It's okay if you do it that way but if so the `elsif` should just be an `else` i.e.
```
else
        posts = posts.order(created_at: :desc)
 end
```
or on the other hand you could do the `elsif` but `orderBy` wouldn't use a default value. 🚀

Reply:
Also worth noting if you go with the else route it should cut down on code for the frontend as well. They would just either pass `orderBy` with true or not pass it at all. Oh also per ruby convention and style guides you should be using snake_case for the argument. The frontend can pass it however but this should be `order_by`. You may need to make sure somewhere in the graphql side that you are using camel case but that is an exception. When writing this method you can call it whatever you like in the arg so following ruby convention would be best.